packages/benchmarks/generator/package.json

```diff
@@ -26,6 +26,7 @@
     "@types/react": "^16.9.36",
     "@types/react-dom": "^16.9.8",
     "@types/rimraf": "^3.0.0",
+    "@web/dev-server": "^0.3.0",
```

Comment:
Maybe it's not really needed, but in the code this replaced `es-dev-server`, should the npm scripts here also be replaced with `wds` and `es-dev-server` dependency removed?

Reply:
Oh! I thought I had done that. I must have somehow reverted it. Great eye!

Reply:
Fixed